integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml

```diff
@@ -142,7 +142,7 @@ spec:
         - name: container-image
           value: "$(tasks.test-metadata.results.container-image)"
         - name: e2e-timeout
-          value: "90m"
+          value: "150m"
```

Comment:
How is this change related to removing the SA?

Reply:
Because the test cases in release-pipelines suite need to run a longer time. Have to change it to make the PR pass.

Reply:
I see it's added now, thanks.